ddtrace/opentelemetry/__init__.py

```diff
@@ -73,13 +73,13 @@ def some_function():
       - ``spanID``
       -
     * - ``trace_state``
-      - ``meta["_sampling_priority_v1"]``, ``meta["_dd.origin"]``, ...
+      - ``meta["tracestate"]["_sampling_priority_v1"]``, ``meta["_dd.origin"]``, ...
```

Comment:
@mabdinur would `meta["_dd.origin"]` also be `meta["tracestate"]["_dd.origin"]` ?

Reply:
Oh no, these are separate tags. meta["tracestate"] is superset of  meta[`_sampling_priority_v1`], meta[`_dd.origin`], meta[`_dd.p.dm`] and tracestate values from third party vendors. 

We can simplify this entry to:

```suggestion
      - ``meta["tracestate"]`` 
```